Packs/Hoxhunt/Integrations/HoxhuntV2/HoxhuntV2.yml

```diff
@@ -0,0 +1,235 @@
+commonfields:
+  id: Hoxhunt v2
+  version: -1
+name: Hoxhunt v2
+display: Hoxhunt v2
+category: Email
+description: "Use the Hoxhunt integration to send feedback to reporters of incidents, set incident sensitivity, and apply SOC classification to incidents."
+configuration:
+- display: Server URL
+  name: url
+  type: 0
+  required: true
+  additionalinfo: e.g. https://api.hoxhunt.com/graphql-external
+  defaultvalue: https://api.hoxhunt.com/graphql-external
+  section: Connect
+- display: API Key
+  name: api_key
+  type: 4
+  required: true
+  additionalinfo: Input your api key from Hoxhunt
+  section: Connect
+- display: Incident Mirroring Direction
+  name: mirror_direction
+  defaultvalue: Incoming And Outgoing
+  type: 15
+  required: false
+  section: Collect
```

Comment:
```suggestion
```
This parameter has 2 times the field `section`, please remove this as it should be in Connect